libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -757,8 +759,18 @@ void AP_CRSF_Telem::process_param_read_frame(ParameterSettingsReadFrame* read_fr
     if (read_frame->destination != 0 && read_frame->destination != AP_RCProtocol_CRSF::CRSF_ADDRESS_FLIGHT_CONTROLLER) {
         return; // request was not for us
     }
+#if AP_CRSF_SCRIPTING
```

Comment:
```suggestion
#if AP_CRSF_SCRIPTING_ENABLED
```

.. and this should be an option in `build_options.py`